app/components/Sidebar/components/Collections.js

```diff
@@ -35,7 +35,7 @@ class Collections extends React.Component<Props> {
   componentDidMount() {
     const { collections } = this.props;
 
-    if (!collections.isFetching && !collections.isLoaded) {
```

Comment:
assuming this check is done internally on the store?

Reply:
`isFetching` is also true when a single collection is fetching, so there was the poss of a race condition which started happening with some of the tweaks on this branch